### PR TITLE
Update the tailwind package to 2.0.3

### PR DESCRIPTION
**Pull request by notjb**
Update the tailwind package to 2.0.3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,17 +29,17 @@
...
   },
   "dependencies": {
     "lodash.max": "^4.0.1",
     "lodash.range": "^3.2.0",
-    "tailwindcss-typography": "^3.0.0"
+    "tailwindcss-typography": ">=3.0.0"
   },
   "peerDependencies": {
-    "tailwindcss": ">=1.4.6"
+    "tailwindcss": ">=2.0.3"
```
Comment (titouanmathis) [inline package.json:42]: We need to keep the v1 constraint along the v2 to be backward compatible and avoid a breaking change. 

```json
  "tailwindcss": ">=1.4.6 || >=2.0.3"
```

**Review by titouanmathis — COMMENTED** [review]
This PR has already been closed? 🤔

**Comment by notjb** [comment]
Hey @titouanmathis 
I deleted the branch and it automatically closed the branch for some reason.
I'm reopening it ;) 
The changes didn't got merged.

**Comment by notjb** [comment]
@titouanmathis the update is now working 🎉 

**Some notes on the specifics :** 
- Didn't update to PostCSS 8 because it is unsupported by VueJS, hence VuePress
- I used instead the compatibility package which comes with TailwindCSS  (https://tailwindcss.com/docs/installation#post-css-7-compatibility-build)
- As of 2.0, tailwind works with node `>=12.3.0` so I configured a `NODE_VERSION` environment variable in netlify to ensure the build process is working

**Review by titouanmathis — APPROVED** [review]
👍  Well done!

**Pull request merged**